mmv1/products/memorystore/Instance.yaml

```diff
@@ -0,0 +1,341 @@
+            name: targetShardCount
+            description: "Output only. Target number of shards for the instance. "
+            output: true
+          - !ruby/object:Api::Type::Integer
+            name: targetReplicaCount
+            description:
+              "Output only. Target number of replica nodes per shard for the
+              instance. "
+            output: true
+        name: updateInfo
+        description: "Represents information about instance with state UPDATING. "
+    name: stateInfo
+    description: "Additional information about the state of the instance. "
+  - !ruby/object:Api::Type::String
+    name: uid
+    description: "Output only. System assigned, unique identifier for the instance. "
+    output: true
+  - !ruby/object:Api::Type::Integer
+    name: replicaCount
+    description:
+      "Optional. Number of replica nodes per shard. If omitted the default
+      is 0 replicas. "
+    default_from_api: true
+  - !ruby/object:Api::Type::String
+    name: authorizationMode
+    description:
+      "Optional. Immutable. Authorization mode of the instance. \n Possible
+      values:\n AUTHORIZATION_MODE_UNSPECIFIED\nAUTH_DISABLED\nIAM_AUTH"
+    immutable: true
+    default_from_api: true
+  - !ruby/object:Api::Type::String
+    name: transitEncryptionMode
+    description:
+      "Optional. Immutable. In-transit encryption mode of the instance. \n
+      Possible values:\n TRANSIT_ENCRYPTION_MODE_UNSPECIFIED\nTRANSIT_ENCRYPTION_DISABLED\nSERVER_AUTHENTICATION"
+    immutable: true
+    default_from_api: true
+  - !ruby/object:Api::Type::Integer
+    name: shardCount
+    description: "Required. Number of shards for the instance. "
+    required: true
+  - !ruby/object:Api::Type::Array
+    name: discoveryEndpoints
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: address
+          description:
+            "Output only. IP address of the exposed endpoint clients connect
+            to. "
+          output: true
+        - !ruby/object:Api::Type::Integer
+          name: port
+          description: "Output only. The port number of the exposed endpoint. "
+          output: true
+        - !ruby/object:Api::Type::String
+          name: network
+          description:
+            "Output only. The network where the IP address of the discovery
+            endpoint will be\nreserved, in the form of\nprojects/{network_project}/global/networks/{network_id}. "
+          output: true
+    description:
+      "Output only. Endpoints clients can connect to the instance through.
+      Currently only one\ndiscovery endpoint is supported. "
+    output: true
+  - !ruby/object:Api::Type::String
+    name: nodeType
+    description:
+      "Optional. Immutable. Machine type for individual nodes of the instance.
+      \n Possible values:\n NODE_TYPE_UNSPECIFIED\nSHARED_CORE_NANO\nHIGHMEM_MEDIUM\nHIGHMEM_XLARGE\nSTANDARD_SMALL"
```

Comment:
Nit: Probably don't need `NODE_TYPE_UNSPECIFIED` listed. Same for other enums.

Reply:
corrected